gradle-plugin/src/main/groovy/io/realm/gradle/RealmTransformer.groovy

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2016 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.gradle
+
+import com.android.build.api.transform.*
```

Comment:
Intellij is was more relaxed than AS when it comes to imports since the JVM is not as a restricted environment as ART or Dalvik. I just went with the default Intellij behaviour here.